torch/distributed/_tensor/op_schema.py

```diff
@@ -334,7 +339,7 @@ class OpInfo:
     flat_args_schema: List[object]
     local_args: Sequence[object]
     local_kwargs: Dict[str, object]
-    args_tree_spec: Optional[TreeSpec] = None
+    args_tree_spec: Optional[PyTreeSpec] = None
```

Comment:
We could leave this as `TreeSpec`. It is aliased to `PyTreeSpec` if `optree` is installed.

```suggestion
    args_tree_spec: Optional[TreeSpec] = None
```